src/main/java/gregtech/common/gui/widget/orefilter/ItemOreFilterTestSlot.java

```diff
@@ -1,111 +1,75 @@
 package gregtech.common.gui.widget.orefilter;
 
-import gregtech.api.gui.IRenderContext;
-import gregtech.api.gui.ingredient.IGhostIngredientTarget;
-import gregtech.api.unification.OreDictUnifier;
-import gregtech.api.util.Position;
+import gregtech.api.util.oreglob.OreGlob;
 
-import net.minecraft.client.Minecraft;
-import net.minecraft.client.renderer.GlStateManager;
-import net.minecraft.client.renderer.RenderHelper;
-import net.minecraft.client.renderer.RenderItem;
-import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.item.ItemStack;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
+import net.minecraftforge.items.ItemStackHandler;
 
-import com.google.common.collect.Lists;
-import mezz.jei.api.gui.IGhostIngredientHandler;
-import org.jetbrains.annotations.NotNull;
+import com.cleanroommc.modularui.widgets.slot.ModularSlot;
 import org.jetbrains.annotations.Nullable;
 
-import java.awt.*;
-import java.util.Collections;
-import java.util.List;
-import java.util.Set;
+public class ItemOreFilterTestSlot extends ModularSlot {
 
-public class ItemOreFilterTestSlot extends OreFilterTestSlot implements IGhostIngredientTarget {
+    OreFilterTestSlot parent;
 
-    @NotNull
-    private ItemStack testStack = ItemStack.EMPTY;
+    OreGlob glob;
 
-    public ItemOreFilterTestSlot(int xPosition, int yPosition) {
-        super(xPosition, yPosition);
+    public ItemOreFilterTestSlot() {
+        super(new ItemStackHandler(1), 0, true);
     }
 
-    @NotNull
-    public ItemStack getTestStack() {
-        return testStack;
+    void setParent(OreFilterTestSlot parent) {
+        this.parent = parent;
     }
 
-    public void setTestStack(@NotNull ItemStack testStack) {
-        this.testStack = testStack;
-        updatePreview();
+    public void setGlob(@Nullable OreGlob glob) {
+        if (this.glob == glob) return;
+        this.glob = glob;
     }
 
-    @Nullable
-    @Override
-    protected Set<String> getTestCandidates() {
-        return this.testStack.isEmpty() ? null : OreDictUnifier.getOreDictionaryNames(this.testStack);
-    }
+    // @Override
+    // @SideOnly(Side.CLIENT)
+    // public boolean mouseClicked(int mouseX, int mouseY, int button) {
+    // if (isMouseOverElement(mouseX, mouseY)) {
+    // EntityPlayer player = Minecraft.getMinecraft().player;
+    // putItem(player.inventory.getItemStack());
+    // return true;
+    // }
+    // return false;
+    // }
```

Comment:
What is the status with this class's commented out code?

Reply:
no longer needed